UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -770,6 +770,11 @@ private List<Color> GetPaletteOrNull()
 	public void UpdateMe()
 	{
 		timeElapsed += UpdateManager.CashedDeltaTime;
+		if (PresentFrame == null)
```

Comment:
is this also related to radiation killing atmos?

Reply:
nah, I just got annoyed with it being spammed on client